scripts/catalog-version.js

```
@@ -0,0 +1,90 @@
+if (!bump || !VALID_BUMPS.has(bump)) {
+  console.error('[catalog-version] Usage: node scripts/catalog-version.js <major|minor|patch>');
+  process.exit(1);
+}
+
+function atomicWrite(filePath, content) {
+  const tmp = filePath + '.tmp';
+  writeFileSync(tmp, content, 'utf8');
+  try {
+    renameSync(tmp, filePath);
+  } catch (e) {
+    try { unlinkSync(tmp); } catch (_) {}
```

ESLint

[error] Empty block statement.

(no-empty)